vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailFragment.kt

```diff
@@ -436,6 +443,26 @@ class RoomDetailFragment @Inject constructor(
         }
     }
 
+    private fun setupRemoveJitsiWidgetView() {
+        views.removeJitsiWidgetView.onCompleteSliding = {
+            withState(roomDetailViewModel) {
+                val jitsiWidgetId = it.jitsiState.widgetId ?: return@withState
+                if (it.jitsiState.hasJoined) {
+                    leaveJitsiConference()
+                }
+                roomDetailViewModel.handle(RoomDetailAction.RemoveWidget(jitsiWidgetId))
+            }
+        }
+    }
+
+    private fun leaveJitsiConference() {
+        JitsiBroadcastEmitter(vectorBaseActivity).emitConferenceEnded()
+    }
+
+    private fun onBroadcastEvent(event: BroadcastEvent) {
```

Comment:
to avoid confusion with the Matrix Events, maybe rename to jitsiEvent or sth like that...